### DISCUSSION TITLE
I cannot access àè my account

### DISCUSSION THREAD

**Discussion by amarroMX** [community/community#23488] [New to GitHub]
<p>Hi everyone,</p>
<p>I want to have access to my previous GitHub account. I registered the account with my university email which has now been deleted. I know the username and password but when I try to connect, there is a device verification step that I cannot complete. Is there a solution to the problem or the account is lost.</p>
<p>Best regards.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>try opening a ticket here, but no assurance</p>
<p><a href="https://support.github.com/contact/account">https://support.github.com/contact/account</a></p>

**Comment by rajeshwar-shinde** [comment]
Hello every one my yahoo account which is linked to old github account is not allowing me to login. So, I want to recover my GitHub account please help.
